v1/media.yaml

```diff
@@ -0,0 +1,284 @@
+swagger: '2.0'
+info:
+  version: '1.0.0-beta'
+  title: MediaWiki page media API
+  description: API for retrieving data about media items appearing on a given page
+  termsOfService: https://www.mediawiki.org/wiki/REST_API#Terms_and_conditions
+  contact:
+    name: Reading Infrastructure
+    url: https://www.mediawiki.org/wiki/Wikimedia_Reading_Infrastructure_team
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+paths:
+  /media/{title}{/revision}:
+    x-route-filters:
+      - path: ./lib/revision_table_access_check_filter.js
+        options:
+          redirect_cache_control: '{{options.response_cache_control}}'
+      - path: lib/security_response_header_filter.js
+        options:
+          allowInlineStyles: true
```

Comment:
We don't need this - it's JSON, not HTML, so no inline styles needed

Reply:
Done. Question: What exactly does the `redirect_cache_control: '{{options.response_cache_control}}'` on line 18 do?

Reply:
That's the cache-control header that will be applied to 301/302 responces